modules/library/coverage/pom.xml

```diff
@@ -122,7 +122,6 @@
   	  <artifactId>imageio-ext-tiff</artifactId>
       <!-- The version number is specified in the parent POM. -->      
     </dependency>
-    
```

Comment:
What happened here? :-D

Reply:
I was getting a transitive dependency POM error so I played a bit with the poms.
looks like the reset to its original version didn't worked fine. Will fix it